.cirrus.yml

```diff
@@ -13,7 +13,15 @@ container:
   image: python:3.8
 
 env:
-  PIP_CACHE_VERSION: "1"
+  # Maximum cache age in weeks before forcing a new cache upload.
+  CACHE_AGE: "2"
```

Comment:
I think this variable would be more accurately described as `CACHE_PERIOD`, as it is, `CACHE_AGE` implies that this describes how old the cache is.